modules/op-using-custom-scc-to-improve-prod-security.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: PROCEDURE
+
+[id='op-using-custom-scc-to-improve-prod-security_{context}']
+= Using a custom SCC to improve pod security
+
+Because you can migrate pipelines that must be run or built as root, you can improve security by using custom security context constraints (SCCs). To lock down your installation further when the default root user is not allowed, allow the container to run only as user ID `1000` when the pod requests what permissions to use.
```

Comment:
The last sentence, do we need it? The only info we need is the benefit of using SCCs so that the user knows if he/she wants to complete this procedure. 

If we can cut it, I say cut it.